A MIME message with multipart/alternative body (text/plain and text/html)
and binary application/octet-stream attachment may be falsely identified as
spam.

There's no text in message body; Razor2::Client::Core::compute_sigs()
should ignore empty text/plain part, but it doesn't because a certain regex
is too strict.

It so happens that that empty body is in Razor database, and is marked as
spam (e4 sig: ui-ZAL0e9rWPUDaa4GWPttamsYEA).
--- a/lib/Razor2/Client/Core.pm
+++ b/lib/Razor2/Client/Core.pm
@@ -560,7 +560,7 @@ sub compute_sigs {
                 $self->log(6,"preproc: mail $objp->{id} went from $olen bytes to 0, erasing");
                 $objp->{skipme} = 1;
                 next;
-            } elsif (($clen < 128) and ($clnpart =~ /^(Content\S*:[^\n]*\n\r?)+(Content\S*:[^\n]*)?\s*$/s)) {
+            } elsif (($clen < 128) and ($clnpart_vr8 =~ /^(Content\S*:.*\n\r?)+(Content\S*:.*)?\s*$/s)) {
                 $self->log(6,"preproc: mail $objp->{id} seems empty, erasing");
                 $objp->{skipme} = 1;
                 next;
